contracts/protocol/lendingpool/LendingPoolConfigurator.sol

```diff
@@ -296,10 +296,10 @@ contract LendingPoolConfigurator is VersionedInitializable, ILendingPoolConfigur
 
       //if threshold * bonus is less than PERCENTAGE_FACTOR, it's guaranteed that at the moment
       //a loan is taken there is enough collateral available to cover the liquidation bonus
-      require(
-        liquidationThreshold.percentMul(liquidationBonus) <= PercentageMath.PERCENTAGE_FACTOR,
-        Errors.LPC_INVALID_CONFIGURATION
-      );
+      // require(
```

Comment:
why did we remove this check?

Reply:
This was removed when we used reserveId = 0 as ithaca collateral, adding it again